dlt/__init__.py

```diff
@@ -62,6 +64,7 @@
     "resource",
     "transformer",
     "defer",
+    "destination",
```

Comment:
please also import `load_package()` into `current` module so it can be requested from within decorated function

Reply:
It is there, no?